libs/langgraph/langgraph/prebuilt/tool_node.py

```diff
@@ -50,6 +51,11 @@
 TOOL_CALL_ERROR_TEMPLATE = "Error: {error}\n Please fix your mistakes."
 
 
+class StateUpdateArtifact(NamedTuple):
```

Comment:
* Does this serialize out of the box?
* Why limit to state_update rather vs. the full GraphCommand?

Reply:
this was before our most recent discussions -- would have to full `GraphCommand`. but yea, NamedTuple should serialize out of box